frontend/src/components/abfragen/bauleitplanverfahren/AllgemeineInformationenZurAbfrageBauleitplanverfahrenComponent.vue

```diff
@@ -47,6 +47,38 @@
         />
       </v-col>
     </v-row>
+    <v-row
```

Comment:
Falls die EAkte editierbar ist wird dieses als Textfeld angezeigt.

Somit kann bei einem User mit editierbarer EAkte der Link nicht geklickt sonder nur kopiert werden.

Mögliche Lösung:

Da es sich um um mehr als ein Textfeld handelt ist diese am besten als eigene Komponente zu erstellen und in die notwendigen Stellen einzubinden.

![image](https://github.com/it-at-m/isi-frontend/assets/56300801/a2e806d9-6f55-40c8-bb72-ba71fb0c3775)